.cargo/config.toml

```diff
@@ -10,16 +10,16 @@ CF_TEST_CONFIG_ROOT = { value = "engine/config/testing", relative = true }
 # `rustup completions --help` to find out how to set this up.
 
 [alias]
-cf-test = "test --features runtime-benchmarks,try-runtime"
-cf-test-ci = "test --release --features runtime-benchmarks,slow-tests,try-runtime"
+cf-test = "test --features runtime-benchmarks,try-runtime,metadata-hash"
+cf-test-ci = "test --release --features runtime-benchmarks,slow-tests,try-runtime,metadata-hash"
 cf-test-cfe = "test --package chainflip-engine --package multisig"
 
-cf-clippy = "clippy --all-targets --features runtime-benchmarks,try-runtime,runtime-integration-tests,slow-tests -- -D warnings -A deprecated"
-cf-clippy-ci = "clippy --all-targets --features runtime-benchmarks,try-runtime,runtime-integration-tests,slow-tests -- -D warnings -A deprecated"
+cf-clippy = "clippy --all-targets --features runtime-benchmarks,try-runtime,runtime-integration-tests,slow-tests,metadata-hash -- -D warnings -A deprecated"
+cf-clippy-ci = "clippy --all-targets --features runtime-benchmarks,try-runtime,runtime-integration-tests,slow-tests,metadata-hash -- -D warnings -A deprecated"
```

Comment:
this was needed because the wasm builder code is activated by both std and metadata-hash feature. But yes, we should decouple them so that we should be able to build without metadata-hash as well. I'll do that and then revert this.